.github/workflows/main.yml

```diff
@@ -17,6 +17,7 @@ jobs:
           - java: 11
             upload_reports: true
           - java: 17
+      fail-fast: false
```

Comment:
What's the purpose of this?

Reply:
If e.g. Java 17 task fails, it won't cancel the Java 11 task. I added it earlier on when we were still using JDK 8 and JDK 11, and there legitimately were reasons why it might fail for one and succeed for the other. I don't know if that's still possible between JDK 11 and JDK 17, but these tasks are relative short (matter of minutes) so if the task is already running why bother cancelling it? Just let it run.